src/main/java/ua/gorshkov/hw19/DBPersonDAO.java

```diff
@@ -0,0 +1,82 @@
+package ua.gorshkov.hw19;
+import org.hibernate.Session;
+import java.util.List;
+
+public class DBPersonDAO implements DAO<Person> {
+    @Override
+    public Person save(Person obj) {
+        try(Session session = MySessionManager.getSession()) {
+            session.beginTransaction();
+            session.save(obj);
+            try {
+                session.getTransaction().commit();
+            } catch (Exception e) {
+                session.getTransaction().rollback();
+                throw new RuntimeException(e);
+            }
+        }
+        return obj;
+    }
+
+    @Override
+    public Person update(Person obj) {
+        try(Session session = MySessionManager.getSession()) {
+            session.beginTransaction();
+            session.update(obj);
+            try {
+                session.getTransaction().commit();
+            } catch (Exception e) {
+                session.getTransaction().rollback();
+                throw new RuntimeException(e);
+            }
+        }
+        return obj;
+    }
+
+    @Override
+    public void delete(Person obj) {
+        try(Session session = MySessionManager.getSession()) {
+            session.beginTransaction();
+            session.delete(obj);
+            try {
+                session.getTransaction().commit();
+            } catch (Exception e) {
+                session.getTransaction().rollback();
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
+    @Override
+    public Person get(Long id) {
```

Comment:
What if we did not find an entity?
Use Optional API to return Optional.empty() when there is no entity and Optional.of(person) when we find it